samples/MvcSample.Web/Startup.cs

```diff
@@ -41,6 +38,9 @@ public void Configure(IApplicationBuilder app)
 
                 app.UseServices(services =>
                 {
+                    services.AddCachingServices();
```

Comment:
nit: Add it at a consistent place, here we are adding it in the beginning while in the else we are adding it after ```AddMvc``` and other calls.